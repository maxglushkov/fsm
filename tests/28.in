L
1
S Sb||a
