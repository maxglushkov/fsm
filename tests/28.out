2
2
SX b SX
S a SX
